dask/dask.yaml

```
@@ -8,6 +8,8 @@ dataframe:
   parquet:
     metadata-task-size-local: 512  # Number of files per local metadata-processing task
     metadata-task-size-remote: 16  # Number of files per remote metadata-processing task
+  assert-eq:
+    scheduler: null  # default scheduler used when computing dataframes for assertions
```

It makes more sense to me to put the default in here since it there are not multiple defaults depending on case.

```suggestion
    scheduler: "sync"  # default scheduler used when computing dataframes for assertions
```